crates/bevy_asset/src/io/mod.rs

```diff
@@ -120,6 +121,38 @@ impl<T: ?Sized + AsyncSeekForward + Unpin> AsyncSeekForward for Box<T> {
     }
 }
 
+pub trait AsyncSeekForwardExt: AsyncSeekForward {
+    fn seek_forward(&mut self, offset: u64) -> SeekForwardFuture<'_, Self>
```

Comment:
Should include some documentation here. E.g.,

```rust
/// Seek by the provided `offset` in the forwards direction, using the [`AsyncSeekForward`] trait.
```

Reply:
Good point